test/integration_test.go

```diff
@@ -72,6 +72,54 @@ func TestCreateResource(t *testing.T) {
 		Done()
 }
 
+func TestUpdateResource(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skpping integratin test in short mode")
+	}
+
+	byt, err := ioutil.ReadFile("../examples/request.json")
+	if err != nil {
+		panic(err)
+	}
+	var postData map[string]interface{}
+
+	if err := json.Unmarshal(byt, &postData); err != nil {
+		panic(err)
+	}
+
+	setupTest().Post("/v1/resource").
+		SetHeader("Foo", "Bar").
+		JSON(postData).
+		Expect(t).
+		Status(201).
+		Type("json").
+		JSONSchema(createSchema).
+		AssertFunc(assertResourceResponse).
+		Done()
+
+	byt, err = ioutil.ReadFile("../examples/update_request.json")
+	if err != nil {
+		panic(err)
+	}
+
+	var patchData map[string]interface{}
+
+	if err := json.Unmarshal(byt, &patchData); err != nil {
+		panic(err)
+	}
+
+	setupTest().Patch(fmt.Sprintf("/v1/resource/%s", id)).
+		SetHeader("Content-Type", "application/json").
+		JSON(patchData).
+		Expect(t).
+		Status(200).
+		Type("json").
+		JSONSchema(createSchema).
+		AssertFunc(assertResourceResponse).
```

Comment:
I'd like to see a GET request here that shows that any changes that were made were actually applied.